bin/rtlcss.js

```diff
@@ -85,7 +85,15 @@ while ((arg = args.shift())) {
       break
     case '-s':
     case '--silent':
-      console.log = console.info = console.warn = function () {}
+      console.info = console.warn = function () {}
+      break
+    case '-x':
+    case '--exclude':
+      exclude = args.shift().split(',').map(function(str) {
```

Comment:
This way, exclude is limited to a set of named files. a glob pattern would be much better (see [glob-fs](https://www.npmjs.com/package/glob-fs))